Dockerfile

```diff
@@ -0,0 +1,15 @@
+# default build is for CentOS7, change the base image to fit your build.
+FROM centos:centos7
+MAINTAINER Sebastien Le Digabel "sledigabel@gmail.com"
+
+RUN yum update -y
+RUN yum install -y git rpmdevtools mock
```

Comment:
Originally I started working on a separate project that would pull the project in the container and build.
Then I thought it would be better if it belonged to the project itself and use volumes.

I'll remove the two packages.

Good catch.